tests/utils/stl/test/executor.py

```diff
@@ -76,12 +76,10 @@ def __init__(self, commandPrefix, chain):
         self.commandPrefix = commandPrefix
         self.chain = chain
 
-    def run(self, exe_path, cmd=None, work_dir='.', file_deps=None, env=None):
-        cmd = cmd or [exe_path]
-        return self.chain.run(exe_path, self.commandPrefix + cmd, work_dir,
+    def run(self, cmd, work_dir='.', file_deps=None, env=None):
+        return self.chain.run([self.commandPrefix] + cmd, work_dir,
                               file_deps, env=env)

```

Comment:
This was just broken.  PostFix looks broken too, but I haven't had a need to use it.